File: app/src/main/java/tech/destinum/machines/data/MachinesDB.java
package tech.destinum.machines.data;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import tech.destinum.machines.data.local.POJO.Income;
import tech.destinum.machines.data.local.POJO.Machine;
import tech.destinum.machines.data.local.dao.IncomeDAO;
import tech.destinum.machines.data.local.dao.MachineDAO;

@Database(entities = {Machine.class, Income.class}, version = 11)
public abstract class MachinesDB extends RoomDatabase{

    public abstract IncomeDAO getIncomeDAO();

    public abstract MachineDAO getMachineDAO();

}

File: app/src/main/java/tech/destinum/machines/Data/MachinesDB.kt
package tech.destinum.machines.Data


import androidx.room.Database
import androidx.room.RoomDatabase
import tech.destinum.machines.Data.Local.Entities.Income
import tech.destinum.machines.Data.Local.Entities.Machine
import tech.destinum.machines.Data.Local.DAOs.IncomeDAO
import tech.destinum.machines.Data.Local.DAOs.MachineDAO

@Database(entities = arrayOf(Machine::class, Income::class), version = 13)
abstract class MachinesDB : RoomDatabase() {

    abstract val incomeDAO: IncomeDAO

    abstract val machineDAO: MachineDAO

}
